### PR TITLE
[new release] dlm (0.3.3)

### PR DESCRIPTION
**Pull request by psafont**
Libdlm bindings

- Project page: <a href="https://github.com/xapi-project/ocaml-dlm/">https://github.com/xapi-project/ocaml-dlm/</a>
- Documentation: <a href="https://xapi-project.github.io/ocaml-dlm/doc">https://xapi-project.github.io/ocaml-dlm/doc</a>

##### CHANGES:

Remove unused "primitive"

The metadata hasn't been maintained in quite some time, so I expect some tweaks

### CODE REVIEW

**Comment by psafont** [comment]
I'm not sure what to do with oraclelinux-8, the os-family seems to be ignored and can't find anything conclusive in the rest of the repo on what should be used. Any ideas?

File: packages/dlm/dlm.0.3.3/opam [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  "unix-errno" {>= "0.5.2"}
+  "uuidm" {>= "0.9.6"}
+]
+depexts: [
+  ["libdlm-dev"] {os-distribution = "debian"}
+  ["libdlm-dev"] {os-distribution = "ubuntu"}
+  ["dlm-devel"] {os-distribution = "rhel"}
+  ["dlm-devel"] {os-distribution = "centos"}
+  ["dlm-devel"] {os-distribution = "fedora"}
+  ["dlm-devel"] {os-family = "oraclelinux"}
```
Comment (mseri) [inline packages/dlm/dlm.0.3.3/opam:31]:
```suggestion
  ["dlm-devel"] {os-family = "ol"}
```

Comment (mseri) [reply packages/dlm/dlm.0.3.3/opam:31]: The values are documented here: https://github.com/ocaml/opam-repository/wiki/Depexts-os-distribution---os-family-values

**Comment by mseri** [comment]
The other issue is due to ocamlbuild 0.9.0 used by unix-errno in a lower-bound. I am sending a fix separately

**Comment by mseri** [comment]
You can ignore the oraclelinux 8 failure, at least now it is trying to install the right thing

**Comment by psafont** [comment]
Not sure why it's not available, the name is correct: https://yum.oracle.com/repo/OracleLinux/OL8/developer/x86_64/getPackage/dlm-devel-4.1.0-1.el8.x86_64.rpm

**Pull request merged**